packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCardBack.tsx

```diff
@@ -17,7 +17,7 @@ const JurisdictionCardBack: FC<TJurisdictionCardBackProps> = ({ setIsFlipped, ve
     const verificationContents = jurisdictionVerificationContents();
     if (!verificationDocs) return <></>;
```

Comment:
If I'm not mistaken it would be better to return`null` rather than an empty fragment. What do you think? (I might be wrong 😅